it/controllers/RegimeAppealsControllerISpec.scala

```diff
@@ -940,6 +935,528 @@ class RegimeAppealsControllerISpec extends IntegrationSpecCommonBase with Regime
+              |    "dateOfAppeal": "2020-01-01T00:00:00",
+              |    "isLPP": false,
+              |    "appealSubmittedBy": "customer",
+              |    "appealInformation": {
+              |						 "reasonableExcuse": "other",
+              |            "honestyDeclaration": true,
+              |            "startDateOfEvent": "2021-04-23T00:00:00",
+              |						 "statement": "This is a statement",
+              |            "lateAppeal": false,
+              |            "uploadedFiles": [
+              |               {
+              |                 "reference":"reference-3000",
+              |                 "fileStatus":"READY",
+              |                 "downloadUrl":"download.file",
+              |                 "uploadDetails": {
+              |                     "fileName":"file1.txt",
+              |                     "fileMimeType":"text/plain",
+              |                     "uploadTimestamp":"2018-04-24T09:30:00",
+              |                     "checksum":"check12345678",
+              |                     "size":987
+              |                 },
+              |                 "uploadFields": {
+              |                     "key": "abcxyz",
+              |                     "x-amz-algorithm" : "AWS4-HMAC-SHA256"
+              |                 },
+              |                 "lastUpdated":"2018-04-24T09:30:00"
+              |               }
+              |            ]
+              |		}
+              |}
+              |""".stripMargin
+          )
+          val expectedJsonResponse: JsObject = Json.obj(
+            "caseId" -> "PR-1234567889",
+            "status" -> MULTI_STATUS,
+            "error" -> "Appeal submitted (case ID: PR-1234567889, correlation ID: uuid-1) but received 500 response from file notification orchestrator"
+          )
+
+          val result = await(buildClientForRequestToApp(uri = s"/$r/appeals/submit-appeal/$it/$i?isLPP=false&penaltyNumber=123456789&correlationId=uuid-1&isMultiAppeal=true").post(
+            jsonToSubmit
+          ))
+
+          result.status shouldBe MULTI_STATUS
+          Json.parse(result.body) shouldBe expectedJsonResponse
+          eventually {
+            wireMockServer.findAll(postRequestedFor(urlEqualTo("/write/audit"))).asScala.toList.exists(_.getBodyAsString.contains("PenaltyAppealFileNotificationStorageFailure")) shouldBe true
+          }
+        }
+      }
+
+      "return BAD_REQUEST (400)" when {
+        "no JSON body is in the request" in {
+          setEnabledFeatureSwitches(CallAPI1808HIP)
+          mockStubResponseForAuthorisedUser
+          val result = await(buildClientForRequestToApp(uri = s"/$r/appeals/submit-appeal/$it/$i?isLPP=true&penaltyNumber=123456789&correlationId=uuid-1").post(
+            ""
+          ))
+          result.status shouldBe BAD_REQUEST
+        }
+
+        "JSON body is present but it can not be parsed to a model" in {
```

Comment:
Can we test for other error cases like if the 'reasonableExcuse' is not one of the valid options, or if it's 'other' and they don't submit data with it?